README.md

```diff
@@ -11,7 +11,6 @@ This project contains the Cyface Android SDK which is used by Cyface application
 How to integrate the SDK
 ---------------------------
 
-- [Submodules](#submodules)
```

Comment:
Because the SDK itself does not have any submodules anymore. The android-utils submodule is now downloaded automatically as maven dependency.

We have the instructions how to integrate the SDK in the frontend Readme (we never ha this in here, see [master](https://github.com/cyface-de/android-backend) branch):

```
## Importing the Cyface SDK, Cyface Android Energy Settings and Cyface Android Camera Service
To get the current submodules' code run the following command:
    $ git submodule update --init
etc ....
```